WBooks/Repositories/BookRepository.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  BookRepository.swift
+//  WBooks
+//
+//  Created by Megan Cole on 28/02/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import Result
+import UIKit
+import Alamofire
+
+internal class BookRepository {
+    public func fetchBooks(onSuccess: @escaping ([Book]) -> Void, onError: @escaping (Error) -> Void) {
+        let url = URL(string: "https://swift-training-backend.herokuapp.com/books")!
```

Comment:
- Move the string to a `static let`. 
- Let's add a `guard` clause to remove the forced unwrapping.